chat-monitor.js

```diff
@@ -3,32 +3,36 @@
 // @namespace      http://somewhatnifty.com
 // @description    reformats twitch chat for display on a chat monitor
 // @match        https://www.twitch.tv/*/chat?display*
-// @version    0.200
+// @version    0.300
 // @updateURL https://raw.githubusercontent.com/paul-lrr/nifty-chat-monitor/master/chat-monitor.js
 // @downloadURL https://raw.githubusercontent.com/paul-lrr/nifty-chat-monitor/master/chat-monitor.js
 // @require  https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js
 // @require  https://gist.github.com/raw/2625891/waitForKeyElements.js
+// @require  https://raw.githubusercontent.com/sizzlemctwizzle/GM_config/master/gm_config.js
 // @grant       GM_getResourceText
 // @grant       GM_addStyle
-// @require  https://raw.githubusercontent.com/sizzlemctwizzle/GM_config/master/gm_config.js
 // @grant       GM_getValue
 // @grant       GM_setValue
 // @grant       GM_log
-// @resource style https://raw.githubusercontent.com/paul-lrr/nifty-chat-monitor/master/chat-monitor.css
+// @resource style http://www.myshelter.net/chat-monitor.css?6
```

Comment:
This should be changed back to the github link. I assume you were testing by putting it on your own server

Reply:
That's right, sorry!